utils/set_env.cmd

```diff
@@ -0,0 +1 @@
+set PYTHONPATH=%PYTHONPATH%;./tabpy-server;./tabpy-tools
```

Comment:
if this was a python script it would be os independent

Reply:
unfortunately it doesn't work all the time with Python script plus sometimes we need this variable outside of test code.